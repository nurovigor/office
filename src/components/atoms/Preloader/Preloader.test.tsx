import React from 'react';
import { render } from '@testing-library/react';
import { Preloader } from './';

describe('Preloader component', () => {
	it('renders Preloader component', () => {
		const { asFragment } = render(<Preloader />);
		expect(asFragment()).toMatchSnapshot();
	});
});
