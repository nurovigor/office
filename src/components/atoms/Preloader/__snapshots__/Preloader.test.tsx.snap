// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader component renders Preloader component 1`] = `
<DocumentFragment>
  <div>
    <div />
  </div>
</DocumentFragment>
`;
